ajax/libs/multi-level-push-menu/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "multi-level-push-menu",
+  "filename": "jquery.multilevelpushmenu.min.js",
+  "author": "Momčilo Džunić <it@make.rs>",
+  "license": "MIT",
+  "homepage": "https://multi-level-push-menu.make.rs",
+  "description": "Cross-browser compatible jQuery plug-in allowing endless nesting of navigation elements.",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/adgsm/multi-level-push-menu.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "jquery.multilevelpushmenu.*",
+          "multilevelpushmenu.jquery.json"
```

Comment:
Actually, we haven't support json files yet, and json files are not needed at most cases, maybe we should remove this unless it's required?

Reply:
sure. I will remove that.

Reply:
Looks like you can do it now

Reply:
so shall I add the json file in the files attribute or not ?